### PR TITLE
theme rebellion fixes for 24.7 new  lobby dashboard

### PR DESCRIPTION
**Pull request by marjohn56**
minor changes and additions to fix display issues

### CODE REVIEW

File: misc/theme-rebellion/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-rebellion
-PLUGIN_VERSION=		1.8.10
+PLUGIN_VERSION=		1.9.1
```
Comment (fichtner) [inline misc/theme-rebellion/Makefile:2]:
```suggestion
PLUGIN_VERSION=		1.9
```

**Comment by fichtner** [comment]
Awesome, merged, thanks!

**Comment by marjohn56** [comment]
Works on mine, but Kev and Ned are checking it to see if there's anything else that needs doing as it's been a while since I looked at it.

**Comment by kozistan** [comment]
Hi guys, thanks for any improve. Just tested, possibly you've updated something on the background, the dashboard is still bright. Anyway "Always Look on the Bright Side of Life" :D

**Comment by marjohn56** [comment]
I'll do a clean install this afternoon and check it out; I think I know what's happening. Watch this space.

**Comment by marjohn56** [comment]
Another PR has been merged which should solve the problems. #4130

**Comment by kozistan** [comment]
Up and working, thanks!

**Pull request merged**